OutOfSchool/OutOfSchool.BusinessLogic/Services/Database/WorkshopService.cs

```diff
@@ -87,7 +88,8 @@ public WorkshopService(
         IRegionAdminService regionAdminService,
         ICodeficatorService codeficatorService,
         ITagService tagService,
-        ISearchStringService searchStringService)
+        ISearchStringService searchStringService,
```

Comment:
Primary constructor?

Reply:
@VadymLevkovskyi We need to have a new separate task for this service refactoring with its 17 dependencies. I remember having some DI issues when using primary constructor

Reply:
Well, I'll be happy to hear about mentioned DI issues.
Regarding refactoring - it has definitely to be done, together with WorkshopServicesCombiner, WorkshopServiceStrategy